assets/css/app.css

```diff
@@ -11,6 +11,21 @@
   width: 3.5rem;
 }
 
+.mw3-5{
```

Comment:
What's the naming convention here?

Reply:
Max width 3.5

Not ideal I know - can you think of a better one? I could make it more custom I guess

Reply:
The book component is using flex-box at the moment, so I guess there should also be a fallback.

Reply:
Sorry, I meant because it was `max-width: 4.2rem;`, I didn't know if the `3-5` was meant to refer to anything else. I think the `x-x` convention is good for decimal numbers though!